website/pages/docs/developers/creating-new-plugin/python-source.mdx

```diff
@@ -1,6 +1,8 @@
 ---
 title: Creating a New CloudQuery Source Plugin in Python
 description: Learn how to write a new CloudQuery source plugin from scratch in Python.
+image: "https://i3.ytimg.com/vi/TSbGHz5Z09M/maxresdefault.jpg"
```

Comment:
What are we expecting these to do?

Reply:
I was expecting it to auto-embed the video, but that didn't work, so I'll follow this up with a second PR that manually adds the embed.

Reply:
https://github.com/cloudquery/cloudquery/pull/14782